src/aics-image-viewer/components/App/index.tsx

```diff
@@ -204,7 +214,10 @@ const App: React.FC<AppProps> = (props) => {
   // Playback state goes here, but the play/pause buttons that mainly control this class are down in `AxisClipSliders`
   const playControls = useConstructor(() => new PlayControls());
   const [playingAxis, setPlayingAxis] = useState<AxisName | "t" | null>(null);
-  playControls.onPlayingAxisChanged = setPlayingAxis;
+  playControls.onPlayingAxisChanged = (axis) => {
+    loader.current?.setPrefetchPriority(axis ? [axisToLoaderPriority[axis]] : []);
+    setPlayingAxis(axis);
+  };
```

Comment:
cool! do you see some performance gain from having prefetched more relevant data?

Reply:
Generally what I've seen is that playback runs at its (relatively low) framerate limit until it catches up with ongoing prefetching, then slows down dramatically. My hope for this change is that it will delay or prevent that slowdown point, though I haven't done a lot of work to test how well it accomplishes that.